internal/wire/wire_test.go

```diff
@@ -118,6 +119,19 @@ func fuzzMessages(f *testing.F, testCases []testCase) {
 		f.Add(tc.expectedB)
 	}
 
+	records, err := loadRecords(filepath.Join("..", "..", "records"))
+	if err != nil {
+		f.Fatal(err)
+	}
+
+	if len(records) == 0 {
+		f.Log("No records found so none of them will be added to the corpus")
+	}
```

Comment:
```suggestion
	f.Logf("%d recorded messages were added to the seed corpus", len(records))
```

So we always log the number

Reply:
Nice idea!